runtime/view.js

```diff
@@ -98,8 +98,18 @@ class View extends ViewBase {
   }
 
   remove(id) {
-    // TODO
+    var trace = tracing.start({cat: "view", name: "View::remove", args: {name: this.name}});
+    if (!this._items.has(id)) {
+      return;
+    }
+    entity = this.items_.get(id);
```

Comment:
i think you meant this._items here, instead of this.items_
probably worth adding a unit test for this exact reason :)